spec/services/events/create_batch_service_spec.rb

```diff
@@ -258,9 +258,9 @@
             timestamp:,
             metadata: {},
           )
-        end.to change { organization.events.count }.by(2)
+        end.to change(Event, :count).by(2)
 
-        expect(organization.events.last.timestamp.iso8601(3)).to eq('2023-09-04T15:45:12.344Z')
+        expect(Event.where(organization_id: organization.id).last.timestamp.iso8601(3)).to eq('2023-09-04T15:45:12.344Z')
```

Comment:
Layout/LineLength: Line is too long. [121/120] (https://rubystyle.guide#max-line-length)